src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,57 @@
+package lotto.domain;
+
+import java.text.MessageFormat;
+import java.util.Objects;
+
+public final class LottoNumber {
+
+    public static final int LOWER_BOUND = 1;
+    public static final int UPPER_BOUND = 45;
+    private final int lottoNumber;
+
+    public LottoNumber(final int value) {
+        validateRange(value);
+        this.lottoNumber = value;
+    }
+
+    public LottoNumber(final String text) {
+        validateBlank(text);
+        lottoNumber = Integer.parseInt(text.trim());
+        validateRange(lottoNumber);
+    }
+
+    private void validateBlank(final String value) {
+        if (value == null || value.isBlank()) {
+            throw new IllegalArgumentException("숫자를 입력해야 합니다");
+        }
+    }
+
+    private void validateRange(final int number) {
+        if (number < LOWER_BOUND || number > UPPER_BOUND) {
```

Comment:
초미세 가독성 팁인데 비교인자를 가운데나 가장자리로 몰아주면 가독성이 좋아집니다 ㅎ
```suggestion
        if (number < LOWER_BOUND || UPPER_BOUND < number) {
```